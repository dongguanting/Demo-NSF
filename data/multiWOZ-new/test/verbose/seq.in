i would like to find a ticket for a train that leaves monday from the station and arrives at destination by 19:00
i will leave from peterborough
i just need one ticket for my trip to cambridge
looking to leave from stansted airport after quarter of twelve because i have a presentation at work that morning
i 'd need to depart after 19:45 from stevenage because that 's closest to where i live
please confirm when the tuesday train leaves
i am willing to go from cabridge to london liverpool street
it does n't really matter but perhaps after 20:15 would be good
i am trying to find out if there are any trains that are going to cambridge
on tuesday i am leaving from broxbourne to cambridge
no i do not know when id like to leave by as long as i get there by 1645
yes i would appreciate if you could book a ticket for me
i need to go to cambridge by train i have to get there at approximately 11:30 can you see the departure time options
im departing from ely does that affect the 11:30 time
i 'm headed back home to leicester from cambridge on saturday to see my family
no  i mean i want leave on the evening 8:45
i am looking for a train departing from london kings cross i need to get to birmingham
on saturday leaving london kings cross and going to cambridge see the available times for me please
the train should be departing from london kings cross and arrive at cambridge by 14:00
that train sounds great please book for 5
i need the tickets asap because i am travelling on monday and need to leave after 16:30
i need to arrive in norwich on friday by 11:15
i need to find a train that will be arriving on wednesday at 12:30
why yesi am departing from birmingham new street and going to cambridge
yes i am needing a train so i will be arriving in cambridge no later than 12:30 please
i 'll have to leave on saturday if i 'm going to make the evening plans
could you tell me if there is a train leaving from cambridge after 17:30
eventually i want to travel everywhere but on saturday i need to get to stansted airport please
i 'm currently searching for trains on monday departing from cambridge
i was wondering if i could get three seats on the next train i 'd really appreciate it
hi there i 'd like to inquire about a train to liverpool street in london please it would need to be leaving after 1715 if possible
i am planning to leave this friday evening
would you please be so kind as to inform me which is the train that departs from cambridge on thursday
my journey will begin in cambridge
yes i need to be at birmingham new street tonight
im going to birmingham new street on monday
only one ticket please
i would prefer to depart from the station located at liverpool street in london england
i want to leave on wednesday to cambridge
is it possible if i can get i do n't know a train that will leave cambridge as soon as monday
thanks for your help ill be leaving kings cross at 5:15
hi i need to leave cambridge by train this sunday coming can you help
the object of my search is the locomotive soujourning to the settlement of stevenage that will arrive by a quarter to 12
trains departing thu for stansted airport during the day
i am wanting to find out about any trains that might be leaving from norwich
hi i need to be in cambridge fairly early on tuesday morning can you help
i will be leaving this wednesdy and i will be leaving from norwich
4 more people are going with me do you also make a reservation for them
hi i want to travel from cambridge to stansted airport could you please give me assistance
yes on sunday 13:45 i want to leave
i 'm traveling alone so i 'll only need one ticket today
actually my train is from cambridge to bishop stortford
please book one ticket only for that train
i am going to london king cross and i have to arrive before 16:45 what are the available times
i want to go to cambridge help me with trains please
i 'm looking for a train on friday from london it will be comfortable that the train leaving from london liverpool street on friday after 20:00
i am looking for a train departing from cambridge to get to birmingham
may in know the details of train  from cambridge on sunday
okay i do n't need directions to kings cross but can you verify the train will arrive in stevenage by 9:00
i am not sure whether i will be going to london liverpool street but i will try and go there on sunday
i want to go to cambridge by train i am looking for some good option
does the train which i was looking going to cambridge depart from broxbourne station
i am hoping to leave this sunday around noon
i want to leave after 13:45 from broxbourne station
i 'm actually not going on vacation next week but please give me more information for a train from cambridge to broxbourne
i need to depart from this station after 19:45 so i need the first train available may i have its id number and the price please
i need to go to stansted airport by train from cambridge station i would like someone to help me get the ticket
i require a train to cambridge
i think it wil be possible for me to depart at 800 to cambridge
that 's good can i get six tickets
i want to find some train help me please - i need cambridge - norwich one
i am going to be leaving on the train on sunday
i just need to be there by 12:30
i 'll take one ticket for that then i 'll be right on time for my next train
yes i want to leave on sunday to cambridge
i think i should leave in the evening let 's try after 17:15
i want 2 so book it
i want to depart from cambridge to new street station in  birmingham new street on monday by 12:45
yes i would like some help which train leaves on thursday and arrived by 9:15
iam leaving for cambridge that leaves on thursday and arrives by 9 :15 where am i now
i need a train for trip going to birmingham new street and i 'm looking to leave on thursday
to birmingham i am leaving from cambridge and i should arrive by 17:15
i want the train that should arrive by 14:45 and should leave on saturday from cambridge to london liverpool
i would like to arrive at 15:45 on tuesday at cambridge
get information on trains out of cambridge this saturday
depending on the weather i want to go to kings lynn
i am in need of a ticket for a train traveling between stansted airport and cambridge station
help me with trains please i need one to ely
i have a doctors appointment wednesday so it will be for thursday
my arrival time is important so i 'm glad you asked and the answer i can give you is that it has to be by 14:15 please
i will be making a journey all the way from cambridge to the town of broxbourne
i would like to leave this saturday around noon
i would like to leave cambridge by 08:30
yes thanks book for 6 people please
i have to be in norwich by midday this coming monday can you arrange a train ticket
i 'd like to travel on a train that will leave after 17:00 and will be traveling to cambridge
can we see if a train will be there by 13:00
i need reservation for 4 people to cambridge
the trains id is 1709 & the departure is on friday
i would like to take a train from cambridge any time after quarter past nine please
helloid like to leave kings lynn by monday and arrive in cambridge by 1445
i am watching for a train to london kings cross that departs after 08:15
help me find some evening train from cambridge perhaps after 20:45
oh i need to the station near liverpool street in london
i have a class wednesday afternoon so thursday is when i want to leave
i am going to take the train this wednesday that leaves specifically after 16:00
i want the train to be arrived by 9:45
ideally id like to leave on sunday
let 's try point of destination birmingham new street and day - wednesday
im booking it for 3 persons
the train goes to stansted airport they have a great restaurant there it should arrive by 21:30
i would like to book a train that leaves cambridge on tuesday and arrives at stansted airport by 21:30
i 'm going to leave on tuesday so i need a train that leaves then and that arrives closest to 21:30 traveling from cabridge to stansted airport
can i please book that train for 8 people and can i also get a reference number
i need to go home so i am trying to find a train
i actually am going on a business travel to london so i 'd like to go to the kings cross station from cambridge
user : i am looking for a train agent : i 'd be happy to help you with that can you tell me your departure location and where you are headed to user : i am looking to go to london kings cross and depart from cambridge agent : there are 70 trains making that trip do you have a day or time
i want to leave from broxbourne after 12:30  on wednesday heading for cambridge
i am trying to search a train that would leave after 15:15 to go to broxbourne station
no i do n't need a train on friday but on sunday afternoon can you find a train at 1515 on sunday
it would be nice if you can help me get train tickets for tuesday
i need to go to cambridge from stevenage after 10:45 in the morning if possible
let 's see if wednesday has a train
i 'd like to book a train to liverpool street in london that leaves after my event at 10
i 'd like a train to london liverpool street that leaves from cambridge after 10:00 on friday please
can you tell me which train is departing from broxbourne
iam going to cambridge departing from broxbourne
how can i reach by 1715 at cambridre
i 'm going to be leaving on wednesday
does that train arrive by 17:15 since i will need to be in cambridge at that time
i need a train to cambridge this saturday i will be departing from london
i am leaving ely this friday preferably in the morning
can we leave at 09:45
i will be departing cambridge at 8:45 on saturday morning
my schedule dictates to leave cambridge  thursday at 08:45  can issue a ticket for that day and time
i need to leave sometime after 20:15 because that 's the earliest i can make it to the station
i 'd like just one seat i 'm taking this train by myself no one is coming with me
if i wanted to travel to cambridge what is the best route to take from norwich that would arrive before noon
i would love to arrive at 14:15 in the lovely area of cambridge not a minute early or late
there are two passengers for that train
please tell me about trains to cambridge that leaaves on friday
hi i 'd like to take a train from cambridge could you help me
i have a flight to mexico from standsted airport can you tell me how i would get there
well i thought about that and the conclusion i 've come to is that thursday would be the best time for me to carry out my travels
i 've got to run a few errands so i need to leave sometime after 16:15
i am looking for reservations for tuesday 's train with tickets for two at 10:30 am with final destination broxbourne and with initial destination in cambridge
actually i am travelling and will be reaching on monday at 14:15
i would prefer to travel sometime on monday any time after 12:00 should work
my destination for monday is leicester
yes i need to go to cambridge monday
just one departing bishops stortford and arriving in cambridge please
well i 'm not leaving now but can i get a ticket fora train that departs from cambridge and arrives in peterborough by 21:45
no i do n't actually need a car but i need a train from london liverpool street station on wednesday
i need to arrive cambrige from peterborough by 14:45
i would prefer a wednesday departure from peterborough
affirmative i like that very much would it be able to book for five different people
i would like to find a train to peterborough that departs from cambridge
i will be traveling to kings lynn this saturday
i am most likely to be leaving from cambridge with kings lynn as the destination in mind
ideally the train should depart for peterborough some time after 10:15
i 'm looking to travel via train from cambridge all the way to ely today
would you be able to help find a train which will arrive in peterborough by 18:30
i am looking to travel from cambridge to peterborough on sunday
i would like more information about trains going to cambridge if possible
i 'm looking for a train that can take me to cambridge my family is there looking forward to see me
the user needs a train that will leave from brookshite
i 'm excited to see more of cambridge can you take me there from london liverpool street please
it will be only myself traveling so all i need is one ticket
could you tell me when the train arrives in london liverpool street from cambridge
no i am looking to leave cambridge on saturday not friday
i am needing a train that will be leaving from cambridge to ely and it should be for this thursday
i need to find a train i am going from cambridge to leicester
i 'm leaving on monday thank you for checking for me
i 'd like to find a train that has the destination to peterborough
would you be able to reserve 6 tickets for the next train after 11:15 on monday
if possible please i would prefer to leave on sunday after 9:00 in the morning
i am going to cambridge thank you for helping me locate the train
i think i 'd like to leave from someplace close how about the london liverpool street
please list of departing trains after 21:00 from london liverpool street
i need to board the train that will leave cambridge and arrive in king lynn at 8am
hi there- how’s it going i’m looking for all outgoing trains that are arriving at cambridge please
well the date will be sunday coming up and i would like to arrive absolutely no later than 13:00
i 'm looking to book a train that will be going to broxbourne please
i need to get to london kings cross so i will need a train to get their on wednesday
i need to leave the station by 18:00 so i can get to my sister 's house in time and eat lunch with them
i am looking for a train running from peterborough to cambridge which arrives no later than 8:45
i would like to leave on saturday from london liverpool street to cambridge
what are the schedules for trains heading from cambridge to ely
i 'll need 3 tickets 1 for me and the other 2 for more 2 people can you book that for 3 people please and if you can get me a reference number as well i 'll thank you even more
i am looking to buy tickets for a train
i 'm currently at bishops stortford and would very much like to go to cambridge please
i will be traveling to cambridge on wednesday departure must be after 08:15
the user is looking for a train that will leave on sunday
the current plan is that we would like to get the travelling done on wednesday
i have an important meeting i need to make so i ca n't arrive any later than 18:15
i will go from cambridge to broxbourne
actually for this moment i am finding a train to take 5me to cambridge
can i get a train that leaves by  9:15 to cambridge from stevenage on thursday
i am leaving the city leicester and i need to leave by the time 20:30
i need to get out so i 'm getting a train ticket out of king lynn on tuesday
i want to depart from kings lynn on tuesday
i 'd like to take the train kings lynn to campbridge on tuesday after 9:30
i need a train going to liverpool street london so i can watch a sports game on saturday
i 'm looking for a train that departs some time after 15:30 and the place i would be leaving from is cambridge
i 'm actually going to beautiful cambridge
i will need to travel to peterborough i am leaving from cambridge
yes please book it for 1 person from peterborough to cambridge and provide the reference number
i am not looking for the train arriving i am  looking for a train leaving after 11:45
on friday i am looking to depart stansted airport for cambridge after 11:45
the user accepted that offer and wants a train for 7 people
i’m departing cambridge on saturday is there trains to birmingham new street after 14:15
i would like information on a train arriving in cambridge some time around about 10:45
i need to take the train from cambridge to norwich
would you be able to tell me if there are any trains departing from cambridge on thursday
i will be traveling from cambridge to broxbourne
if there is train will arrive close to 15 the that 's okay
i would like tickets from cambridge to broxbourne for 6 people
do you by any chance know where i can find a train to peterborough from cambridge ideally by this sunday
i would like to book a train leaving cambridge after 18:15
is there a departure from norwich to cambridge on saturday
i will be departing from norwich after 14:30
i am looking to book tickets for a train departing from cambridge at 9:15 or later
hey i am looking for a train to go to cambridge and i need to travel on thursday from london liverpool street
i need to get on a train to cambridge
actually  am looking for the train comes from kings lynn
in terms of the arrival i 'm looking to get there on the thursday
i would like to depart from kings lynn after 10:00 thursday
i am searching for the right train for a trip i have in mind
okay thursday is the day i 'd like to leave and cambridge is where i would like to leave from
please find me all the trains that go to peterborough
it is in my best interest to go from cambridge to peterborough and i hope that i am able to leave after 2:15 in the afternoon
it is on tuesday that i will be traveling
i am in need of a train going from stevenage to cambridge
are there any trains leaving from stevenage on teusday and arriving to cambridge by 9:00
i want to leave for cambridge
i will arrive from london kings cross
i need a train with the departure being from london and the destination i 'm looking to arrive at is kings cross
the user wants a train that will arrive in cambridge by 17:00 on a tuesday
the user will leave to cambridge and he needs a train with his specific informations of time and place
if possible i 'm wondering if there happens to be something that might be able to depart a little earlier around 20:30
i 'd like sunday
yes can you check if you have place for a ticket for 8 people please
i would like to buy two tickets from kings lynn to cambridge
the 17:36 should work does that train leave on friday
thanks for taking the time to let me know the price and also the reference number huge help
hello can you please help me get a train from ely to camebridge thank you
i will be traveling on friday and would like to arrive in cambridge by 14:15
the user says he wants to go from a stevenage from cambridge
yes i would like to book it for 8 people from stevenage to cambridge and i will need the reference number
i am looking to book a train departing from stevenage arriving at cambridge on saturday
i think the best time for me that i want to leave is any time after 12:15
i do n't need to the train number but i 'll be there by 20:45
i am looking to place a reservation for a train arriving by 19:00 friday
i will be leaving from the cambridge station on friday
my arriving destination on friday will be stansted airport
the train should go to cambridge and should depart from london liverpool street does it have
the user requested a train that will take him/her from cambridge to broxbourne
the train will be taking me from cambridge to broxbourne at 8:15 on monday
can i get a train from cambridge to bishops stortford
i 'd like to leave from cambridge on thursday after 20:15
good evening i wonder if you would be able to help me find a train that departs cambridge anytime after 21:00 precisely this coming thursday
i am seeking more information re: saturday trains to cambridge
of course do you have in saturday
i 'm not sure i want to go but by 13:45
i need a train that leaves on wednesday with an arrival of 12:30
trains are frustrating but i 'm traveling from cambridge to london liverpool street
the user would like a train that leaves for leicester at tuesday so that 's what he asks
i am looking for a train departing leicester after 11:15 on tuesday
the destination is indeed cambridge and in terms of timing i need to depart after 11:15 from my departure point of leicester
i would very much like to find a train departing from leicester on saturday
i want to go the british city called cambridge and i have to be there after 12 post meridiem
i am looking for a train from london kings cross and leaves on wednesday
on friday i desire to travel from london liverpool street by means of a train
i am looking for a train which arrives at cambridge by 14:00 or earlier
user: i 'm actually looking for a train so i can get to cambridge in time by living norwich now
i would like to leave from norwich to cambridge on tuesday and arrive by 13:30
the user is looking for a train that leaves on a sunday and after 21:45
i need to go to stansted airport from  cambridge
i would prefer to leave cambridge sometime after 21:45
i’m departing london kings cross after 19:00 on sunday can i receive help booking a train from cambridge
at this time i will need tickets for the six people who will be traveling on the train
i would like to request more information about trains leaving london kings cross and also arriving by 14:30 at the latest
my destination on thursday will be cambridge
that works for me and i will go ahead and get you to make a booking for 7 people nearest time slot to 10 would be ideal
i need to leave from cambridge thanks to the nice weather
i would like to leave from cambridge on a thursday
as long as i 'm able to get off the train and arrive at my destination by 17:45 that 's all that matters
i would like to catch a train thats leaving from cambridge on the coming sunday
as long as the train ends up in leicester we 're golden
i will want to leave after 10:15 because it will be better for me
i am searching for a train departing bishops stortford and arriving at cambridge
i plan to leave this saturday at any time available
could you tell me also about sunday anyways by 8:15
i 'm in need for a ticket to a train from london kings cross to cambridge
are there trains scheduled to run on sunday
can you help me find a train leaving from stevenage to see my grandmother
is there a train from stevenage to cambridge on friday
i will want to leave a little after 1:30 pm to get there on time
