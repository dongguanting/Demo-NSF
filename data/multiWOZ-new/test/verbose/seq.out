O O O O O O O O O O O O B-day O O O O O O O O B-arrive
O O O O B-depart
O O O B-people O O O O O O
O O O O B-depart I-depart O B-leave I-leave I-leave O O O O O O O O O
O O O O O O B-leave O B-depart O O O O O O O O
O O O O B-day O O
O O O O O O O O B-depart I-depart I-depart
O O O O O O O O B-leave O O O
O O O O O O O O O O O O O O O B-dest
O B-day O O O O B-depart O O
O O O O O O O O O O O O O O O O O O B-arrive
O O O O O O O O O O O O
O O O O O B-dest O O O O O O O O O B-arrive O O O O O O O
O O O B-depart O O O O O O
O O O O O O B-dest O B-depart O B-day O O O O
O O O O O O O O B-leave I-leave
O O O O O O O O B-depart I-depart I-depart O O O O O O
O B-day O O O O O O O B-dest O O O O O O O
O O O O O O O O O O O O B-dest O B-arrive
O O O O O O O B-people
O O O O O O O O O O B-day O O O O O B-leave
O O O O O O O O O B-arrive
O O O O O O O O O O O B-day O B-arrive
O O O O O B-depart I-depart I-depart O O O B-dest
O O O O O O O O O O O O B-dest O O O B-arrive O
O O O O O O B-day O O O O O O O O O
O O O O O O O O O O O B-depart O B-leave
O O O O O O O O B-day O O O O O B-dest I-dest O
O O O O O O O B-day O O B-depart
O O O O O O O B-people O O O O O O O O O O
O O O O O O O O O O O B-dest I-dest I-dest I-dest O O O O O O O O B-leave O O
O O O O O O B-day O
O O O O O O O O O O O O O O O O O B-depart O B-day
O O O O O B-depart
O O O O O O B-dest I-dest I-dest O
O O O O O O O B-day
O B-people O O
O O O O O O O O O O B-depart I-depart O I-depart O
O O O O O B-day O O
O O O O O O O O O O O O O O O O B-depart O O O B-day
O O O O O O O B-dest I-dest O B-leave
O O O O O B-depart O O O B-day O O O O
O O O O O O O O O O O O O B-dest O O O O B-arrive I-arrive I-arrive I-arrive
O O B-day O B-dest I-dest O O O
O O O O O O O O O O O O O O B-depart
O O O O O O B-dest O O O B-day O O O O
O O O O O B-day O O O O O O B-depart
B-people O O O O O O O O O O O O O O
O O O O O O B-depart O B-dest I-dest O O O O O O
O O O B-leave O O O O
O O O O O O O O O B-people O O
O O O O O B-depart O B-dest I-dest
O O B-people O O O O O
O O O O B-dest I-dest I-dest O O O O O O B-arrive O O O O O
O O O O O B-dest O O O O O
O O O O O O O B-day O O O O O O O O O O O B-depart I-depart I-depart O B-day O B-leave
O O O O O O O O B-depart O O O O
O O O O O O O O B-depart O B-day
O O O O O O O O O O O O O O O O O O B-dest O O
O O O O O O O O O O B-dest I-dest I-dest O O O O O O O O O
O O O O O B-dest O O O O O O O O O
O O O O O O O O O O O O B-depart O
O O O O O O B-day O O
O O O O B-leave I-leave O O O
O O O O O O O O O O O O O O O O O O O B-depart O B-dest
O O O O O O O O B-leave O O O O O O O O O O O O O O O O O
O O O O O B-dest I-dest O O O B-depart O O O O O O O O O O O
O O O O O B-dest
O O O O O O O O O O O B-leave O B-dest
O O O O O O B-people O
O O O O O O O O O O O O B-depart O B-dest O
O O O O O O O O O O B-day
O O O O O O O B-arrive
O O O B-people O O O O O O O O O O O O O O
O O O O O O B-day O O
O O O O O O O O O O O O B-leave
O O B-people O O O
O O O O O B-depart O O O O O B-dest I-dest I-dest O B-day O B-arrive
O O O O O O O O O O B-day O O O B-arrive
O O O B-depart O O O O O O O O O O O O O
O O O O O O O O B-dest I-dest I-dest O O O O O O O B-day
O O O O O O B-depart O O O O O B-arrive
O O O O O O O O O O O O O O O B-depart O O O
O O O O O O O O B-day O O
O O O O O O B-depart O B-day
O O O O O O O O O B-dest I-dest
O O O O O O O O O O O O B-depart I-depart O B-dest O
O O O O O O O O O B-dest
O O O O O O O O O O O B-day
O O O O O O O O O O O O O O O O O O O O O O O O O B-arrive O
O O O O O O O O O O B-dest O O O O O
O O O O O O B-day O O
O O O O O O O B-leave
O O O O B-people O O
O O O O O B-depart O O O O B-day O O O O O O
O O O O O O O O O O O O B-leave O O O O O B-dest
O O O O O O O O O O B-arrive
O O O O B-people O O O
O O O O O O O O O O B-day
O O O O O O O O B-depart O O O B-leave I-leave I-leave O
O O O O B-depart I-depart O B-day O O O B-dest O B-arrive
O O O O O O O B-dest I-dest I-dest O O O B-leave
O O O O O O O B-depart O O B-leave
O O O O O O O B-dest I-dest I-dest I-dest
O O O O O O O B-day O O O O O O
O O O O O O O O B-day O O O B-leave I-leave
O O O O O O O O B-arrive
O O O O O O B-day
O O O O O O B-dest I-dest I-dest O O O B-day
O O O O B-people O
O O O O B-dest I-dest O O O O O O O O O O B-arrive
O O O O O O O O O O O B-day O O O O O O O
O O O O O O B-day O O O O O O O O O O O O O B-arrive O O O O B-dest I-dest
O O O O O O O B-people O O O O O O O O O
O O O O O O O O O O O O B-depart
O O O O O O O O O B-dest O O O O O O O O B-dest I-dest O O B-depart
O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O O
O O O O O B-depart O B-leave O O O O O
O O O O O O O O O O O B-leave O O O B-dest O
O O O O O O O O O O O B-day O O O O O O O O O O
O O O O O O O O O O O O O B-day
O O O O O B-dest O B-depart O B-leave O O O O O
O O O O B-day O O O
O O O O O O O O B-dest I-dest I-dest B-dest O O O O O O B-leave
O O O O O O O O O O O O B-depart O O O B-day O
O O O O O O O O O B-depart
O O O B-dest O O O
O O O O O B-arrive O O
O O O O O O O B-day
O O O O O B-arrive O O O O O O O B-dest O O O
O O O O O B-dest O O O O O O O O
O O O B-depart O B-day O O O O
O O O O B-leave
O O O O O O B-leave O B-day O
O O O O O O B-day O O O O O O O O O O O
O O O O O O B-leave O O O O O O O O O O O O
O O O O B-people O O O O O O O O O O O O O O
O O O O O O B-dest O O O O O O O O B-depart O O O O O
O O O O O O B-arrive O O O O O O O O O O O O
O O B-people O O O O
O O O O O O B-dest O O O B-day
O O O O O O O O O B-depart O O O O
O O O O O O O B-dest I-dest O O O O O O O O O
O O O O O O O O O O O O O O B-day O O O O O O O O O O O O
O O O O O O O O O O O O O O O B-leave
O O O O O O B-day O O O O O B-people O B-arrive O O O O B-dest O O O O O B-depart
O O O O O O O O O B-day O B-arrive
O O O O O O O B-day O O O B-leave O O
O O O O O B-dest
O O O O O O B-dest B-day
O B-people O O O O O O O O
O O O O O O O O O O O O O O O O O B-depart O O O B-dest O B-arrive
O O O O O O O O O O O O O O B-depart I-depart I-depart O O B-day
O O O O O O O O B-arrive
O O O O B-day O O O
O O O O O O O O O O O O O B-people O O
O O O O O O O O B-dest O O O B-depart
O O O O O B-dest I-dest O O
O O O O O O O O B-depart O O O O O O O O
O O O O O O B-dest O O O B-leave
O O O O O O O O B-depart O O O O B-dest O
O O O O O O O O O O O O O B-dest O B-arrive
O O O O O O B-depart O O O B-day
O O O O O O O O O B-dest O O
O O O O O O O O O O O B-dest O O O O O O O O O
O O O O O O O O O B-depart
O O O O O O O B-dest O O O O O O B-depart I-depart I-depart O
O O O O O O O O O O O B-people O
O O O O O O O O O B-dest I-dest I-dest O O
O O O O O O O O B-day O O
O O O O O O O O O O B-depart O B-dest O O O O O O B-day
O O O O O O O O O O B-depart O B-dest
O O O O B-day O O O O O O
O O O O O O O O O O O O B-dest
O O O O O O B-people O O O O O O B-leave O O
O O O O O O O O O B-day B-leave I-leave O O O
O O O O B-dest O O O O O O O O
O O O O O O O O O O O O O B-depart I-depart I-depart
O O O O O O B-leave O O O O
O O O O O O O O O B-depart O O O B-dest I-dest O O
O O O O O O O O O O O O O O O B-depart O
O O O O O B-day O O O O O O O O O O O O B-arrive
O O O O O O O O O O O O B-dest O
O O O O O B-dest I-dest I-dest O O O O O O O O O O B-day
O O O O O O O B-arrive O O O O O O O O O O O O O O O O
O O O O O O O O B-depart O B-dest O O O O O B-arrive
O O O O O O B-day O O O O O O
O O O O O O O O B-depart O B-dest
O O O B-people O O O O O O O O O O O O O O O O O B-people O O O O O O O O O O O O O O O O O O O
O O O O O O O O B-dest
O O O O B-depart I-depart O O O O O O O O B-dest O
O O O O O O O B-day O O O O B-leave
O O O O O O O O O O O B-day
O O O O O O O O O O O O O O B-day
O O O O O O O O O O O O O O O O O B-arrive
O O O O B-depart O B-dest
O O O O O O O O O O O O O B-dest
O O O O O O O O O O O O B-depart O B-day
O O O O O B-depart O O O O O O O O B-leave
O O O O O O O O O O O O O O B-depart I-depart O B-day
O O O O O O O O B-day
O O O O O O O B-depart I-depart O B-dest O B-day O O
O O O O O O B-dest I-dest I-dest O O O O O O O O B-day
O O O O O O O O O O O B-leave O O O O O O O O O B-depart
O O O O O O B-dest
O O O O O O B-dest O O O O B-depart
O O O O O B-people O O O O O O O O O O
O O O O O O O O O O O O O O O O B-leave
O B-day O O O O O B-depart I-depart O B-dest O O
O O O O O O O O O O B-people O
O O O O O O O O O B-dest I-dest I-dest O B-leave
O O O O O O O O O B-dest O O O O B-arrive
O O O O O O O B-depart O B-dest
O O O O O O O O O O O O O O B-depart O B-day
O O O O O O O B-dest
O O O O O O O O B-arrive O O O O
O O O O O O O O O B-people O
O O O O O O O O O O O O O B-dest O B-depart O O O B-day
O O O O O O O O O O B-leave
O O O O O B-depart O O O O
O O O O O O O B-leave
O O O O O O O O O O O B-depart O O O O
O O O O O O O O O O B-dest O O O O O O B-day O B-depart I-depart I-depart
O O O O O O O O B-dest
O O O O O O O O B-depart I-depart
O O O O O O O O O O O O O B-day
O O O O O O O O O B-leave O
O O O O O O B-depart O O O O O O O
O B-day O O O O O O O O O B-depart O O O O O O O O
O O O O O O O O O B-dest
O O O O O O O O O B-depart O O O O O O O O O O O O B-leave I-leave I-leave I-leave
O O O B-day O O O O O
O O O O O O O O O B-depart O B-dest
O O O O O O O O B-day O O O O O B-arrive
O O O O O B-dest
O O O O B-depart I-depart I-depart
O O O O O O O O O B-depart O O O O O O O O O O I-depart O
O O O O O O O O O B-dest O B-arrive O O B-day
O O O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O O O O O O B-leave
O O O B-day
O O O O O O O O O O O O B-people O O
O O O O O B-people O O O O O O
O O O O O O O O O B-day
O O O O O O O O O O O O O O O O O O
O O O O O O O O O O B-depart O B-dest O O
O O O O O B-day O O O O O O O O B-arrive
O O O O O O O O O B-depart O O
O O O O O O O O B-people O O O O O O O O O O O O
O O O O O O O O O B-depart O O O O B-day
O O O O O O O O O O O O O O O O B-leave
O O O O O O O O O O O O O O B-arrive
O O O O O O O O O O O O B-arrive B-day
O O O O O O B-depart O O O
O O O O O O O B-dest I-dest
O O O O O B-dest O O O O B-depart I-depart I-depart O O O
O O O O O O O O O O B-depart O B-dest
O O O O O O O O O O O O O B-day
O O O O O O B-depart O B-dest I-dest
O O O O O O O O B-day O B-leave
O O O O O O O O O O O O O O O O O B-depart O O B-leave O O O B-day
O O O O O O B-day O O B-dest
O O O O O O B-day
O O O O O O O O O O B-arrive
O O O O O O O B-day O O O O B-arrive
O O O O O O O O B-depart O B-dest I-dest I-dest
O O O O O O O O O B-depart O B-day O O O O O O
O O O O O O O O O B-leave O O
O O O O B-dest O O O O O O O O O O O O O O O O O
O O O O O O O O O O O O O B-day
O O O O O O O O B-dest O O O O O O O B-leave I-leave I-leave
O O O O O O O B-depart I-depart I-depart O O O B-day
O B-day O O O O O B-depart I-depart I-depart O O O O O
O O O O O O O O O O O B-arrive O O
O O O O O O O O O O O O O B-dest O O O O B-depart O
O O O O O O O O O O B-day O O O B-arrive
O O O O O O O O O O O B-day O O B-leave
O O O O O B-dest I-dest O B-depart
O O O O O O O O B-leave
O O B-depart I-depart I-depart O B-leave O O O O O O O O O O O
O O O O O O O O O B-people O O O O O O O O
O O O O O O O O O O B-depart I-depart I-depart O O O O B-arrive O O O
O O O O O O B-dest
O O O O O O O O O O O O O O O O O B-people O O O O O B-leave O O O
O O O O O B-depart O O O O O
O O O O O O O O O B-day
O O O O O O O O O O O O O O O O O B-arrive O O O O O
O O O O O O O O O O B-depart O O O B-day
O O O O O O O O B-dest O O O
O O O O O O B-leave O O O O O O O
O O O O O O O B-depart I-depart O O O B-dest
O O O O O B-day O O O O
O O O O O O O O O B-arrive
O O O O O O O O O O O B-depart I-depart I-depart O B-dest
O O O O O O O B-day
O O O O O O O O O B-depart O O O O
O O O O O O O B-dest O B-day
O O O O O O O O B-leave I-leave O O O O O
